package.json

```diff
@@ -17,6 +17,8 @@
   },
   "homepage": "https://github.com/liatrio/libotrio#readme",
   "dependencies": {
-    "@slack/bolt": "^3.4.0"
+    "@slack/bolt": "^3.4.0",
+    "dotenv": "^10.0.0",
```

Comment:
do we still need this if you are populating the env variables with docker-compose?